api/routes/system.py

```
@@ -0,0 +1,1224 @@
+                        "error": "Process not running"
+                    })
+            except psutil.NoSuchProcess:
+                ping_results.append({
+                    "worker_type": worker_type,
+                    "pid": pid,
+                    "status": "ping_failed",
+                    "error": "Process not found"
+                })
+            except Exception as e:
+                ping_results.append({
+                    "worker_id": worker_id,
```

Ruff (0.12.2)

Undefined name `worker_id`

(F821)